packages/next/server/web/sandbox/readable-stream.ts

```diff
@@ -59,7 +59,20 @@ class ReadableStream<T> {
       }
     }
 
-    pull()
+    function startPull() {
```

Comment:
I'd name it something like "inject pull" or "register pull" because it doesn't actually start to pull.